packages/fxa-shared/email/emailValidatorErrors.js

```diff
@@ -0,0 +1,28 @@
+const dns = require('dns');
```

Comment:
New files, wherever possible, should be TypeScript. Was it not compiling properly in this location when using a `.ts` file?

Reply:
updated